app-ios/Modules/Sources/Timetable/TimetableDayHeader.swift

```diff
@@ -8,40 +8,88 @@ struct TimetableDayHeader: View {
+                ForEach(
+                    [DroidKaigi2023Day].fromKotlinArray(DroidKaigi2023Day.values()),
+                    id: \.ordinal
+                ) { (day: DroidKaigi2023Day) in
+                    Button {
+                        onSelect(day)
+                    } label: {
+                        VStack(spacing: 0) {
+                            Text(day.name)
+                                .font(Font.system(size: 12, weight: .semibold))
+                            Text("\(day.dayOfMonth)")
+                                .font(Font.system(size: 24, weight: .semibold))
+                                .frame(height: 32)
+                        }
+                        .padding(4)
+                        .frame(maxWidth: .infinity)
+                        .foregroundStyle(
+                            selectedDay == day
+                            ? AssetColors.Primary.onPrimary.swiftUIColor
+                            : AssetColors.Surface.onSurfaceVariant.swiftUIColor)
                     }
-                    .padding(4)
-                    .frame(maxWidth: .infinity)
-                    .foregroundStyle(
-                        selectedDay == day
-                        ? AssetColors.Primary.onPrimary.swiftUIColor
-                        : AssetColors.Surface.onSurfaceVariant.swiftUIColor)
-                    .background(
-                        selectedDay == day
-                        ? AssetColors.Primary.primary.swiftUIColor
-                        : Color.clear
-                    )
-                    .clipShape(Capsule())
                 }
             }
+            .padding(.horizontal, 16)
+            .padding(.vertical, 12)
+            .background(alignment: .center) {
+                GeometryReader { geometry in
+                    Capsule()
+                        .fill(AssetColors.Primary.primary.swiftUIColor)
+                        .frame(width: calculateButtonWidth(deviceWidth: geometry.size.width), height: 56)
+                        .offset(x: calculateDynamicTabHorizontalOffset(deviceWidth: geometry.size.width), y: 10)
+                        .animation(.easeInOut(duration: 0.16), value: selectedDay)
+                }
+            }
+        }
+    }
+
+    private func calculateDynamicTabHorizontalOffset(deviceWidth: CGFloat) -> CGFloat {
+        let buttonAreaWidth = calculateButtonWidth(deviceWidth: deviceWidth)
+        // Get the index value corresponding to `selectedDay` and use it for calculation
+        let indexBySelectedDay = getIndexBySelectedDay()
+        return buttonAreaLeadingMargin + (betweenButtonMargin + buttonAreaWidth) * CGFloat(indexBySelectedDay)
+    }
+
+    private func getIndexBySelectedDay() -> Int {
+        switch selectedDay {
+        case .day1:
+            return 0
+        case .day2:
+            return 1
+        case .day3:
+            return 2
+        default:
+            return 1
         }
```

Comment:
How about use `ordinal` property? 👀 
It might be the same as current one.

Reply:
Oh, this is exactly what I wanted to check if we can use or not! 👍

Reply:
@ry-itto 

Thank you for review! I try to use `ordinal` property.
It seems to fix `getIndexBySelectedDay()` as below👍 

```swift
private func getIndexBySelectedDay() -> Int {
    Int(selectedDay.ordinal)
}
```